SoftLayer/CLI/virt/upgrade.py

```diff
@@ -20,15 +22,17 @@
               help="CPU core will be on a dedicated host server.")
 @click.option('--memory', type=virt.MEM_TYPE, help="Memory in megabytes")
 @click.option('--network', type=click.INT, help="Network port speed in Mbps")
+@click.option('--add', type=click.INT, required=False, help="add Hard disk in GB")
+@click.option('--disk', nargs=1, help="update the number and capacity in GB Hard disk, E.G {'number':2,'capacity':100}")
```

Comment:
Have this be a Tuple instead of a json string. https://click.palletsprojects.com/en/7.x/options/#tuples-as-multi-value-options. And change the name to --resize-disk to make it clear what it does.

@click.option('--resize-disk', type=(int, int), help="Update disk number to size in GB. --resize-disk 1 250 ")